archivist_samples/software_bill_of_materials/README.md

```diff
@@ -54,28 +40,26 @@ This Python class makes it easy to manage SBOM distribution in RKVST and publish
 To create a brand new SBOM Asset and begin tracking and sharing the release history of a product line, use `SoftwarePackage.create()`:
 
 ```python
-# Essential imports
-from archivist import archivist
-from software_package import SoftwarePackage
```

Comment:
This is still needed....?  SoftwarePackage

Reply:
Yes but I figured I'd shorten the thing to just the 'interesting' line of code. Lots of things are needed that I didn't include, so I figured it's better to leave the basic furniture and stuff to the reader rather than showing a small slice of it